src/Data/Cache.hs

```diff
@@ -121,15 +121,15 @@ withAssocCache body = evalStateT (body assocCache) []
 -- testing
 -- testing
 -- [3,3,3,3,7,7]
--- 
+--
 -- >>> withAssocCache $ withFiniteCache (-1) $ testC
 -- one
 -- two
 -- testing
 -- [3,3,3,3,7,7]
 finiteCache :: Monad m => Int -> Cache m k a -> Cache (StateT Int m) k a
 finiteCache n c = Cache
```

Comment:
This is my fault, I did not pay attention to the original alignment.
I corrected the alignment now.